app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class LikesController < ApplicationController
+  def new
+    @like = Like.create(user_id: params[:user_id], post_id: params[:post_id])
+    redirect_to(user_post_path(@like.user, @like.post))
+  end
```

Comment:
- [ ] @mahammad-mostafa Great work on your application, I loved the implementation of the milestone requirements, keep up the great work, however, its good practice to create both fail and success paths in create, update, and delete methods to handle errors in the app effectively, kindly consider adding failure and success path for the new method as you did in comments and posts controller.